### PR TITLE
release 6.2.0 changelog

### PR DESCRIPTION
**Pull request by c2thorn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 ## 6.2.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_certificate_manager_certificates` ([#19361](https://github.com/hashicorp/terraform-provider-google/pull/19361))
+* **New Resource:** `google_network_security_server_tls_policy` ([#19314](https://github.com/hashicorp/terraform-provider-google/pull/19314))
+* **New Resource:** `google_scc_v2_folder_scc_big_query_exports` ([#19327](https://github.com/hashicorp/terraform-provider-google/pull/19327))
+* **New Resource:** `google_scc_v2_project_scc_big_query_exports` ([#19311](https://github.com/hashicorp/terraform-provider-google/pull/19311))
```
Comment (SarahFrench) [inline CHANGELOG.md:7]: Shouldn't these resources have been named `google_scc_v2_folder_scc_big_query_export` and `google_scc_v2_project_scc_big_query_export` instead? Here's a [link to the API docs](https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.bigQueryExports/get) where a singular resource is referred to as an export.

Sorry to bring this up in a CHANGELOG review, but I think this is a mistake we have an opportunity to correct

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+IMPROVEMENTS:
+* assuredworkload: added field `partner_service_billing_account` to `google_assured_workloads_workload` ([#19358](https://github.com/hashicorp/terraform-provider-google/pull/19358))
+* bigtable: added support for `column_family.type` in `google_bigtable_table` ([#19302](https://github.com/hashicorp/terraform-provider-google/pull/19302))
+* cloudrun: added support for nfs and csi volumes (for cloud storage fuse) for google_cloud_run_service ([#19359](https://github.com/hashicorp/terraform-provider-google/pull/19359))
```
Comment (SarahFrench) [inline CHANGELOG.md:12]:
```suggestion
* cloudrun: promoted support for nfs and csi volumes (for Cloud Storage FUSE) for `google_cloud_run_service` to GA ([#19359](https://github.com/hashicorp/terraform-provider-google/pull/19359))
```

The original PR looks like its a beta=>GA promotion, not adding a new field

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+* cloudrunv2: added support for nfs and gcs volumes for google_cloud_run_v2_job ([#19359](https://github.com/hashicorp/terraform-provider-google/pull/19359))
```
Comment (SarahFrench) [inline CHANGELOG.md:13]:
```suggestion
* cloudrunv2: promoted support for nfs and gcs volumes for `google_cloud_run_v2_job` to GA ([#19359](https://github.com/hashicorp/terraform-provider-google/pull/19359))
```

File: CHANGELOG.md (hunk 1 above)
Comment (melinath) [reply CHANGELOG.md:7]: +1 good catch!

Comment (c2thorn) [reply CHANGELOG.md]: they've been renamed, cherrypicked, and this has been updated

**Pull request merged**